lib/screens/home/explorer/staking_modal/staking_modal.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:provenance_wallet/common/pw_design.dart';
+import 'package:provenance_wallet/screens/home/explorer/staking_modal/staking_initial.dart';
+import 'package:provenance_wallet/screens/home/explorer/staking_modal/staking_modal_bloc.dart';
+import 'package:provenance_wallet/services/account_service/transaction_handler.dart';
+import 'package:provenance_wallet/services/models/account_details.dart';
+import 'package:provenance_wallet/services/models/commission.dart';
+import 'package:provenance_wallet/services/models/delegation.dart';
+import 'package:provenance_wallet/services/models/detailed_validator.dart';
+import 'package:provenance_wallet/util/get.dart';
+
+class StakingModal extends StatefulWidget {
```

Comment:
If this is an entire screen, should this be `StakingScreen` to keep with convention?